### ISSUE TITLE
Sing up button need some improvements to remove duplicate information 

### ISSUE THREAD

**Issue by christosptr** [mr-bobz/talarts#133]
Remove the aria-label and change the title of the link
Currert HTML
<a aria-label="Opens New Window" target="_blank" class="btn btn-custom-black" title="Sign Up Now (opens new window to 3rd party site)" href="http://visitor.r20.constantcontact.com/d.jsp?llr=67lifhpab&amp;p=oi&amp;m=1115858153373&amp;sit=lxc8balib&amp;f=b119c4d1-f7ba-41c3-a67b-2afc56cca4fd">Sign Up Now</a>

Solution  
<a href="http://visitor.r20.constantcontact.com/d.jsp?llr=67lifhpab&amp;p=oi&amp;m=1115858153373&amp;sit=lxc8balib&amp;f=b119c4d1-f7ba-41c3-a67b-2afc56cca4fd" title="opens new window to 3rd party site" class="btn btn-custom-black" target="_blank" >Sign Up Now</a>

**Comment by mr-bobz** [comment]
Removed aria-label. On the title, I suggest you and @HerinHentry discuss first and come to a conclusion, I just changed yesterday night :-)
https://github.com/mr-bobz/talarts/issues/108

**Comment by HerinHentry** [comment]
Different screen reader and browser combinations announce the title and aria label differently.

We need to do some more analysis before avtioning this item.

Some do not announce title and some do not announce aria labels. 

Cheers, 
Herin 

Sent from my phone

<div>-------- Original message --------</div><div>From: Christos Petrou notifications@github.com </div><div>Date:07/11/2015  5:04 PM  (GMT+10:00) </div><div>To: mr-bobz/talarts talarts@noreply.github.com </div><div>Cc:  </div><div>Subject: [talarts] Sing up button need some improvements to remove duplicate
  information  (#133) </div><div>
</div>Remove the aria-label and change the title of the link
Currert HTML
<a aria-label="Opens New Window" target="_blank" class="btn btn-custom-black" title="Sign Up Now (opens new window to 3rd party site)" href="http://visitor.r20.constantcontact.com/d.jsp?llr=67lifhpab&amp;p=oi&amp;m=1115858153373&amp;sit=lxc8balib&amp;f=b119c4d1-f7ba-41c3-a67b-2afc56cca4fd">Sign Up Now

Solution

<a href="http://visitor.r20.constantcontact.com/d.jsp?llr=67lifhpab&amp;p=oi&amp;m=1115858153373&amp;sit=lxc8balib&amp;f=b119c4d1-f7ba-41c3-a67b-2afc56cca4fd" title="opens new window to 3rd party site" class="btn btn-custom-black" target="_blank" >Sign Up Now

**Comment by christosptr** [comment]
JAWS announce both and is quite destructing. What does NVDA do

**Comment by HerinHentry** [comment]
Hi @christosptr & @mr-bobz : I am testing with NVDA 2015 + Firefox combination.
The current setting is read as 
        Sign Up Now **(Anchor text)**
        visited  link
         Sign Up Now (opens new window to 3rd party site) **(Title element)**

When tested with NVDA + IE, the current setting is read as 
        Sign Up Now
        link
**Title is not announced**

If we change as per @christosptr 's suggestion this will work fine with Firefox not with IE. Duplicating issue can be a problem for now. But, when the Assistive technologies updates the duplicates in future releases, this code will be future proof. I think we should check with Steve on what he thinks about this link.

**Comment by christosptr** [comment]
ok lets leave it as is then.

My thought though is that we should be developing correctly and not doing work around.
When browsers and screen readers fix their issues we will be left with broken websites

**Comment by christosptr** [comment]
@HerinHentry Can you please close this defect if there is not other changes you would like to happen

**Comment by HerinHentry** [comment]
@christosptr : I think this issue can go to steve for confirmation. As he is a screen reader user, he will be able to comment on the behaviour.

**Comment by christosptr** [comment]
will ask

**Comment by christosptr** [comment]
message send on slack

**Comment by christosptr** [comment]
Hi @HerinHentry  I have tested this with NVDA and it looks good to me. NVDA says the following
Sign Up Now
link  Sign Up Now (opens new window to 3rd party site)

Can you please have a look at this and close

**Comment by HerinHentry** [comment]
Firefox + NVDA combination says both Aria-label and the Title. It seems repeated. But, atleast it's accessible. We can close this issue.

opens new window to 3rd party site
link  Sign Up Now (opens new window to 3rd party site)
